source/api/presentation/3.0/index.md

```diff
@@ -1411,6 +1411,33 @@ Annotation Collections _MUST_ have a URI, and it _SHOULD_ be an HTTP(S) URI. The
 }
 ```
 
+For Annotation Collections with many Annotations, there will be many pages. The Annotation Collection refers to the first and last page, and then the pages refer to the previous and next pages in the ordered list.  Each page is part of the Annotation Collection.
```

Comment:
_"pages refer to the previous and next pages in the ordered list"_... is not borne out by the example. Should we say "The example below shows the first page in a list, and thus has a`next` but no `previous` property."?

Reply:
The example immediately before this shows `AnnotationCollection` with `first` and `last`, then this is an example of one page, so I think it's OK in context.

Reply:
OK, that makes sense